server/server.js

```diff
@@ -14,10 +14,14 @@ dotenv.config({path: path.join(__dirname, './config/config.env' )})
 
 console.log('request came in here!');
 
-app.use((req, res, next) => {
-    res.header('Access-Control-Allow-Origin', '*');
-    next();
-  });
+app.all('*', function(req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*')
+  res.header('Access-Control-Allow-Methods', 'PUT, GET, POST, DELETE')
+  res.header('Access-Control-Allow-Headers', 'Content-type, Authorization')
+  next()
+})
+
+app.use(express.static('/home/grosso63/my-app/build'))
```

Comment:
Is this a local file outside of our repository?

Reply:
replaced with path function and relative path